sass/components/_barrier.scss

```diff
@@ -123,26 +125,54 @@
 /*hover*/
 
 .price {
+    display: block;
     font-size: 12px;
     font-weight: bold;
-    display: block;
-    padding: 3px 1px;
     line-height: 18px;
-    overflow: hidden;
+    overflow: unset;
+    padding: 3px 1px;
+    position: relative;
+    right: 45px;
     text-align: center;
-}
+    width: max-content;
 
-.draggable .price {
-    padding-left: 16px;
+    &--zero {
+        color: $color-blue;
+        @include themify($themes) {
+            background-color: themed('DefaultBg');
+        }
+    }
+    &-overlay {
+        background-color: $color-blue;
+        height: 24px;
+        opacity: 0.3;
+        position: relative;
+        right: -10px;
+        top: -12px;
+    }
+}
 
-    &:after {
-        font-weight: normal;
-        content: '≡';
```

Comment:
Ahh, so you replaced this line with <div> x3? Nice)

Reply:
Thanks, yes, the option with symbol is kind of old way and it was causing issues with moving the value out as it was a part of it